RELEASENOTES.md

```diff
@@ -2,3 +2,4 @@
 - Add `--wait` option to `migrate-repo` command. If set to `true` (default is `false`) it will synchronously wait for the migration to finish, otherwise it will just queue up a repo migration and return the `migration-id`.
 - Support parallel migrations with `generate-script` for both `ado2gh` and `gh gei`. `generate-script` now by default generates a script to perform migrations in parallel. Adding `--sequential` flag will force migrations to perform in a sequential (one by one) fashion.
 - Deprecate `--ssh` flag in `generate-script` and `migrate-repo` commands for both `ado2gh` and `gh gei`.
+- GraphQL API calls now support getting paginated results.
```

Comment:
I would remove this bullet. This is an implementation detail that our users shouldn't need to know or care about IMO